gulpfile.js

```diff
@@ -91,37 +91,47 @@ function writeServiceWorkerFile() {
     cacheId: 'polymerproject',
     staticFileGlobs: [
       `${rootDir}/1.0/index.html`,
```

Comment:
Maybe add the 404 page as a dynamicUrl (since we'll probably always want to have that available?) and the rest (index, about) to the runtime caching patterns?